src/scene/skin-partition.js

```diff
@@ -385,8 +384,6 @@ pc.extend(pc, function () {
                             }
                         }
                     }
-
-                    base += partition.indexCount;

```

Comment:
did someone intend to use this for something?  Maybe a mistake?

Reply:
I think originally, base was set as on the base property in the primitive instead of 0. It's probably not needed though so I think it's fine to delete.